### PR TITLE
fix JUnit version property usage

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by brendandburns** [comment]
This `pom.xml` is regenerated whenever we generate the code. Can you add it to .openapi-generator-ignore, so that it doesn't get overwritten next time?

**Comment by sullis** [comment]
> This `pom.xml` is regenerated whenever we generate the code. Can you add it to .openapi-generator-ignore, so that it doesn't get overwritten next time?

Done.

File: kubernetes/.openapi-generator-ignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 *sbt*
 git_push.sh
 README.md
+pom.xml
```
Comment (yue9944882) [inline kubernetes/.openapi-generator-ignore:7]: the default behavior of the underlying code-generator is to omit the pom.xml file when it's already present, so adding this the list won't break anything..

**Review by yue9944882 — COMMENTED** [review]
/lgtm

File: kubernetes/.openapi-generator-ignore (hunk 1 above)
Comment (brendandburns) [reply kubernetes/.openapi-generator-ignore:7]: oops, my bad. I asked him to do this...

**Pull request merged**